extensions/amp-ad/amp-ad.md

```diff
@@ -135,6 +135,7 @@ When using `amp-iframe` approach:
 
 
 ## Running ads from a custom domain
+*This approach to load/ enhance ads has been deprecated and will be unsupported starting Apr,2018 . Please use [RTC](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/rtc-documentation.md) as an alternative.*
```

Comment:
My understanding is that the current deprecation plan is only for DoubleClick to remove remote.html support. I don't think we want to add this here.